tests/benchmark_sse1.py

```diff
@@ -3,15 +3,13 @@
 # University of Illinois/NCSA Open Source License.  Both these licenses can be
 # found in the LICENSE file.
 
-import subprocess, tempfile, os, sys, shutil, json
-from subprocess import Popen, PIPE, STDOUT
+import tempfile, os, sys, shutil, json, re
 
 __rootpath__ = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 def path_from_root(*pathelems):
   return os.path.join(__rootpath__, *pathelems)
 sys.path += [path_from_root('')]
-import tools.shared
-from tools.shared import *
+from tools.shared import Popen, PYTHON, WINDOWS, CLANG_CPP, EMCC, get_clang_native_args, get_clang_native_env, PIPE
```

Comment:
Maybe it could be worth keeping a <80 character line length, at least for the new code and when it makes sense.

So that this doesn't need to be fixed in the future to make flake8 happy.